webserver/views/datasets.py

```diff
@@ -41,8 +41,13 @@ def eval_jobs(dataset_id):
     ds = get_dataset(dataset_id)
     jobs = db.dataset_eval.get_jobs_for_dataset(ds["id"])
     # TODO(roman): Remove unused data ("confusion_matrix", "dataset_id").
+    last_edited_time = ds["last_edited"]
     for job in jobs:
         if "result" in job and job["result"]:
+	    cmp_time = max(job["created"],last_edited_time)
+            job["valid"] = '0'
+	    if(cmp_time == job["created"]):
+		job["valid"] = '1'
```

Comment:
After looking at this statement again, it could be written more clearly. How about
```python
job["outdated"] = False
if last_edited_time > job["created"]:
    job["outdated"] = True
```

or even

```python
job["outdated"] = last_edited_time > job["created"]
```